src/main/webapp/script.js

```diff
@@ -13,8 +13,11 @@ function createTable() { // eslint-disable-line no-unused-vars
     columnOne.innerText = 'RunID';
```

Comment:
If you look at my current PR I condensed this into a function and an array to make it a lot more concise and less redundant, consider that.

Reply:
+1.
Also give a meaningful name for the column, e.g. runIdColumn instead columnOne, etc.